src/Controller/AdminController.php

```diff
@@ -34,6 +34,7 @@ protected function isActionAllowed($actionName)
         switch ($actionName) {
             // autocomplete action is mapped to list action for access permissions
```

Comment:
the comment is clear ;-)

Reply:
see the list does not necessarily require the same permissions as see the autocomplete data

Reply:
@alterphp They are different

Reply:
Why not, we all have different ways to structure role hierarchies. This is fortunately customizable to your needs by overriding `isActionAllowed` method.